sdk/digitaltwins/azure-digitaltwins-core/review/azure-digitaltwins-core_python.json

```diff
@@ -0,0 +1 @@
twin_id"}, {"Kind": 3, "Value": ":"}, {"Kind": 2, "Value": " "}, {"Kind": 6, "Value": "str"}, {"Kind": 3, "Value": ""}, {"Kind": 3, "Value": ","}, {"Kind": 2, "Value": " "}, {"Kind": 1, "Value": ""}, {"Kind": 0, "Value": "            "}, {"Kind": 5, "DefinitionId": "azure.digitaltwins.core.aio.DigitalTwinsClient.upsert_relationship:async.param(relationship_id", "Value": ""}, {"Kind": 0, "DefinitionId": "azure.digitaltwins.core.aio.DigitalTwinsClient.upsert_relationship:async.param(relationship_id", "Value": "relationship_id"}, {"Kind": 3, "Value": ":"}, {"Kind": 2, "Value": " "}, {"Kind": 6, "Value": "str"}, {"Kind": 3, "Value": ""}, {"Kind": 3, "Value": ","}, {"Kind": 2, "Value": " "}, {"Kind": 1, "Value": ""}, {"Kind": 0, "Value": "            "}, {"Kind": 5, "DefinitionId": "azure.digitaltwins.core.aio.DigitalTwinsClient.upsert_relationship:async.param(relationship", "Value": ""}, {"Kind": 0, "DefinitionId": "azure.digitaltwins.core.aio.DigitalTwinsClient.upsert_relationship:async.param(relationship", "Value": "relationship"}, {"Kind": 3, "Value": ":"}, {"Kind": 2, "Value": " "}, {"Kind": 6, "Value": "Dict"}, {"Kind": 3, "Value": "["}, {"Kind": 6, "Value": "str"}, {"Kind": 3, "Value": ", "}, {"Kind": 6, "Value": "object"}, {"Kind": 3, "Value": "]"}, {"Kind": 2, "Value": " "}, {"Kind": 3, "Value": "="}, {"Kind": 2, "Value": " "}, {"Kind": 9, "Value": "None"}, {"Kind": 3, "Value": ","}, {"Kind": 2, "Value": " "}, {"Kind": 1, "Value": ""}, {"Kind": 0, "Value": "            "}, {"Kind": 5, "DefinitionId": "azure.digitaltwins.core.aio.DigitalTwinsClient.upsert_relationship:async.param(**kwargs", "Value": ""}, {"Kind": 0, "DefinitionId": "azure.digitaltwins.core.aio.DigitalTwinsClient.upsert_relationship:async.param(**kwargs", "Value": "**kwargs"}, {"Kind": 1, "Value": ""}, {"Kind": 0, "Value": "        "}, {"Kind": 3, "Value": ")"}, {"Kind": 2, "Value": " "}, {"Kind": 3, "Value": "->"}, {"Kind": 2, "Value": " "}, {"Kind": 5, "DefinitionId": "azure.digitaltwins.core.aio.DigitalTwinsClient.upsert_relationship:async.returntype", "Value": ""}, {"Kind": 6, "Value": "Dict"}, {"Kind": 3, "Value": "["}, {"Kind": 6, "Value": "str"}, {"Kind": 3, "Value": ", "}, {"Kind": 6, "Value": "object"}, {"Kind": 3, "Value": "]"}, {"Kind": 1, "Value": ""}, {"Kind": 2, "Value": " "}, {"Kind": 1, "Value": ""}, {"Kind": 1, "Value": ""}, {"Kind": 2, "Value": " "}, {"Kind": 1, "Value": ""}], "Diagnostics": [{"DiagnosticId": "AZ_PY_1", "Text": "Typehint is missing for method __init__", "TargetId": "azure.digitaltwins.core.DigitalTwinsModelData.__init__"}, {"DiagnosticId": "AZ_PY_2", "Text": "Typehint is missing for method __init__", "TargetId": "azure.digitaltwins.core.EventRoute.__init__"}, {"DiagnosticId": "AZ_PY_3", "Text": "Typehint is missing for method __init__", "TargetId": "azure.digitaltwins.core.IncomingRelationship.__init__"}, {"DiagnosticId": "AZ_PY_4", "Text": "Typehint is missing for method __init__", "TargetId": "azure.digitaltwins.core.QueryResult.__init__"}]}
```

Comment:
We can probably remove this file now that we have completed the review :)

Reply:
Removed :)